### ISSUE TITLE
"You are currently running Duplicati unknown"

### ISSUE THREAD

**Issue by bauerjj** [duplicati/duplicati#3383]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: unknown
- **Operating system**: Ubuntu 16.04
- **Backend**: B2Blaze

## Description

Go to "About" and click "check for updates now". Version reads "unknown" 

## Steps to reproduce

Go to "About" and click "check for updates now". Version reads "unknown"

**Comment by warwickmm** [comment]
I am unable to reproduce this.  Do you remember what version you installed, and how you installed it?

**Comment by ts678** [comment]
Version data might also be in  About --> System info, About --> Changelog, or various changelog.txt

I'm not super familiar with this UI, but for me the About --> General page gives a "Currently running" automatically. I think I might have seen an unknown briefly awhile ago which soon updated properly.

"Check for updates now" is on the next line. If "currently running" was after clicking, what is above it? Basically, the topic title doesn't need a click, yet the details call for one (which then fails). Confusing...

**Comment by bauerjj** [comment]
@ts678 @warwickmm Thanks - I looked at the Changelog tab and found this:
`2018-04-02 - 2.0.3.3_beta_2018-04-02`

Checking for updates doesn't yield any new information. 

I was using duplicati just fine for over a month. Just yesterday I noticed that when I click "Edit Configuration', the past config details have all reset, yet I can still run my backup tasks just fine and upload the encrypted files to the cloud. The config details in the past used to auto-populate to what I previously set them. This is what forced me to look at the software version and attempt to update. I don't know if this is related to the problem or not. I can supply logs if that helps. I unfortunately don't have a straightforward, reproducible process since I just spontaneously noticed this.

**Comment by warwickmm** [comment]
That's strange.  If you go to your backup, then Advanced -> Database, is the "Local database path" field populated?

**Comment by bauerjj** [comment]
The database is valid and I'm able to open up the 32MB file in sqlitebrowser and examine the contents. The AES passphrase is retained along with the paths of the source data, however the destination metadata (hosting provider, buckets, passphrase, etc) are missing in Duplicate browser. I am unable to select any storage type actually in the dropdown. The "Encryption" dropdown under "General" is also not populated correctly and I can't select any option - it should read "aes". I can't seem to find this meta data saved in the sqlite database file either. 

I exported the configuration and everything in it is valid, however the "encryption-module" is not being populated correctly by the browser from the json. Where is the Destination meta-data being saved?

**Comment by ts678** [comment]
Duplicati-server.sqlite because it's relevant to the server. From its size (and how you found it via the job), I suspect you're seeing the job database. [Developer documentation](https://github.com/duplicati/duplicati/wiki/Developer-Documentation) explains some of the architecture. Other suggestions I offer would be to use the Commandline option of the job to see whether things get pulled in (some also come from the job database), or to try an Export to do the same test. There must be some parts of Duplicati that can get everything together, or I'd expect your backup would be failing. Are you sure this isn't just a web UI problem? Try reloads (the harder the better), cache clear, different browser? The web UI does have a lot of code that runs in the browser, so please change anything that might interfere with that.

**Comment by bauerjj** [comment]
@ts678 Thanks for your help. I opened the sqlite db and verified most of its content to be true. 

 I upgraded from Ubuntu 16.04 to Ubuntu 18 yesterday. I launched duplicati to verify if the problem still persisted, and it did not. The browser displayed the correct history of each of my jobs and the version now reads correctly: `You are currently running Duplicati - 2.0.3.3_beta_2018-04-02`. I'm sorry that I couldn't find exactly what went wrong.
